src/styles/blocks/card.scss

```diff
@@ -0,0 +1,123 @@
+:root {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+  background-color: gray;
+}
+
+.card {
+  @include flex-column;
+
+  box-sizing: border-box;
+  background-color: white;
+  width: 200px;
+  height: 408px;
+  padding-top: 32px;
+  padding-bottom: 16px;
+  border: 1px solid #F3F3F3;
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)